README.md

```diff
@@ -846,7 +846,7 @@ And this set of workflow options:
 
 ```json
 {
-  "defaultRuntimeOptions": {
+  "default_runtime_options": {
```

Comment:
Glad this change is documented, we'll need to make sure to communicate it to our Green friends.